src/server/middleware/sendHtml.js

```diff
@@ -57,42 +58,70 @@ export function safeSend(res, ...payload) {
   }
 }
 
-export function renderStaticErrorPage(res) {
+let errorPage;
+
+export async function setErrorPage(fallbackUrl) {
+  try {
+    const timeoutFetch = createTimeoutFetch(6e3)(fetch);
+    const response = await timeoutFetch(fallbackUrl);
+    const contentType = response.headers.get('content-type');
+    const contentLength = response.headers.get('content-length');
+
```

Comment:
maybe check the fetch call was successful ?

Reply:
@JAdshead if it isn't a 200 response, what do you suggest that we do?

Reply:
this needs to handle that scenario, you don't want to be setting the domains 404/500 error page to be this error page. `response.text()` won't throw for those scenarios and if it does the error will be misleading

Reply:
Fixed in https://github.com/americanexpress/one-app/pull/281/commits/5c5d4dabcee6e57da852f32b2eb4b7bc519f9d89